app/pages/Accounts/useAccountSync.ts

```diff
@@ -21,38 +20,28 @@ const accountInfoUpdateInterval = 30000;
 
 export const accountInfoFailedMessage = (message: string) =>
     `Failed to load account information from your connected node due to: ${message}`;
-const updateTransactionsFailedMessage = (message: string) =>
-    `Failed to load transactions from external service due to: ${message}`;
 
 /**
- * Keeps account info and transactions for selected account in sync. Is dependant on a full re-mount when chosen account changes.
+ * Ensures that the account info (from the node) and the list of transactions (from the wallet proxy),
+ * for the selected account, are kept synchronized. Is dependant on a full re-mount when the
+ * chosen account changes.
```

Comment:
I don't know if this is a reasonable condition (or whether it was ever satisfied)

Reply:
Seems it is (because it is only used directly by withAccountSync)